plane/source/docs/soaring.rst

```diff
@@ -205,7 +208,9 @@ Limit maximum distance from home
    Available in firmware revisions 4.1 and later.
 
 
-If using FBWB or CRUISE mode, the parameter :ref:`SOAR_MAX_RADIUS<SOAR_MAX_RADIUS>` can be used to trigger RTL if the aircraft is more than this distance from home when thernalling in LOITER mode. Note that this parameter won't stop the aircraft from exceeding this distance before it enters LOITER mode.
+If using FBWB or CRUISE mode, the parameter :ref:`SOAR_ALT_CUTOFF<SOAR_ALT_CUTOFF>` can be used to trigger RTL if the aircraft is more than this distance from home when thernalling while in LOITER mode. Note that this parameter won't stop the aircraft from exceeding this distance before it enters LOITER mode.
+
+If Soaring is enabled when beyond this :ref:`SOAR_ALT_CUTOFF<SOAR_ALT_CUTOFF>`, it will not begin any Soaring operations.
```

Comment:
Should be SOAR_MAX_RADIUS. More specifically, it will not switch mode into LOITER automatically - it will still do throttle suppression etc.

Reply:
copy paste errors....sorry...
but if enabled beyond the radius, will it RTL? or just begin gliding until MIN is reached and THEN trigger RTL?

Reply:
fixed